openshift/rh-che.config.yaml

```diff
@@ -41,7 +41,6 @@ data:
   che-system-admin-name: "admin"
   che-system-super-privileged-mode: "true"
   che-websocket-endpoint: "wss://che.prod-preview.openshift.io/api/websocket"
-  che-wsagent-cors-allowed-origins: "*"
```

Comment:
Have we finished checks that everything works without CORS?

Reply:
Katka done some testing: https://github.com/redhat-developer/rh-che/issues/1116#issuecomment-446896060

Reply:
Thanks for the details!